clients/java/client/src/test/java/org/camunda/bpm/client/impl/ExternalTaskClientBuilderImplTest.java

```diff
@@ -91,4 +94,17 @@ public String getBaseUrl() {
     }
   }
 
+  @Test
+  public void testWithNullBaseUrl() {
+    // given
+    ExternalTaskClientBuilderImpl builder = new ExternalTaskClientBuilderImpl();
+
+    // when building with null base url then exception is thrown
+    assertThat(builder.getBaseUrl()).isNull();
+    assertThatThrownBy(builder::build)
+        .isInstanceOf(ExternalTaskClientException.class)
+        .hasMessageContaining("Base URL cannot be null or an empty string");
+
```

Comment:
Nit: I think this part is already tested by `ClientIt.shouldThrowExceptionDueToBaseUrlIsNull`